.pre-commit-config.yaml

```diff
@@ -21,6 +21,8 @@ repos:
         -   ruamel.yaml==0.18.6
         -   questionary==2.0.1
         -   types-colorama==0.4.15.20240311
+        -   types-Pygments==2.18.0.20240506
+        -   types-python-dateutil==2.9.0.20241003
```

Comment:
Why not `datetime.strptime` (https://stackoverflow.com/a/2265383) to avoid an additional dependency?

Also, this/these additional (optional) dependenc(y/ies) should be mentioned in other places, I think.